tethysext/atcore/tests/integrated_tests/models/app_users/spatial_resource_tests.py

```diff
@@ -156,3 +157,12 @@ def test_get_extent_invalid_type(self):
             resource.get_extent('bad_type')
         self.assertTrue('"bad_type" is not a valid type for a SpatialResource '
                         'extent. Must be one of "wkt", "geojson", or "dict"' in str(exc.exception))
+
+    def test_update_extent_srid(self):
+        """Test trying to get an extent of an invalid type throws the correct error."""
+        resource = self.create_resource_in_session()
+        resource.extent = self.expected_geometry
+        resource.update_extent_srid(3857)
```

Comment:
Add an assert before you call `update_extent_srid` to verify that the SRID is not already 3857 (ie that it is actually changed).

Reply:
We manually set it to 4326 (see line 40). There was no SRID before that.

Reply:
This is how we define the wkt. so if SRID=3857, it has changed.
self.extent_wkt = 'SRID=4326;POLYGON((-109.557466506958 40.4626541436629,-109.557123184204 40.4481880854746,' \
                          '-109.528369903564 40.4484166930046,-109.528756141663 40.4629153531036,-109.557466506958 ' \
                          '40.4626541436629))'